templates/init/dataconnect/schema.gql

```diff
@@ -1,28 +1,49 @@
-# # Example schema for simple email app
-# type User @table(key: "uid") {
-# 	uid: String!
-# 	name: String!
-# 	address: String!
+# # Example schema for simple movie review app
+
+# # Users
+# # Suppose a user can leave reviews for movies
+# # user  -> reviews is a one to many relationship,
+# # movie -> reviews is a one to many relationship
+# # movie <-> user is a many to many relationship
+# type User
+#   @table {
+#   id: String! @col(name: "user_auth")
+#   username: String! @col(name: "username", dataType: "varchar(50)")
+#   # The following are generated from the @ref in the Review table
```

Comment:
But the code does not contain `@ref` literally. Shall we reword this? e.g. "generated by the `user: User!` field in the Review table"